Description: verbose error message when the serie file does not exist
 Displays an error message when the $SERIES file does not exist and when run
 in verbose mode.
Bug-Debian: http://bugs.debian.org/369908 
Upstream-status: submitted 2012-12-19

---
 quilt/scripts/patchfns.in |    3 +++
 1 file changed, 3 insertions(+)

--- a/quilt/scripts/patchfns.in
+++ b/quilt/scripts/patchfns.in
@@ -323,6 +323,9 @@ cat_series()
 		sed -e '/^#/d' -e 's/^[ '$'\t'']*//' \
 		    -e 's/[ '$'\t''].*//' -e '/^$/d' $SERIES
 	else
+		if [ "$opt_verbose" ]; then
+			echo "series file not found: $SERIES" >&2
+		fi
 		return 1
 	fi
 }
